composer.json

```diff
@@ -21,7 +21,7 @@
         }
     },
     "require": {
-        "php": ">=7.4",
+        "php": ">=8.0",
```

Comment:
Why do we need to bump php version? won't this reduce number of potential places where this can be used?

Reply:
This was because I introduced use of the function "str_contains". I've changed the code to `strpos` and reverted to 7.4. 

7.4 is almost EOL (https://www.php.net/supported-versions.php) but it is still shipped with Debian 11. 

It's not a very big deal because people can install an older release of Uplink-PHP, composer is smart enough to select the correct version.